src/test/java/com/google/sps/MealServletTest.java

```diff
@@ -154,6 +154,7 @@ public void getMealListTest() throws IOException, ServletException {
         MealServlet servlet = new MealServlet();
         MockHttpServletRequest request = new MockHttpServletRequest();
         MockHttpServletResponse response = new MockHttpServletResponse();
+        request.setPathInfo("/query=");
```

Comment:
In the test, to simulate `/meal?query=aaa`, do `setPathInfo("/")` and `addParameter("query","aaa")`.